Gemfile.lock

```diff
@@ -131,6 +133,7 @@ DEPENDENCIES
   rubocop (~> 1.25.0)
   rubocop-rake
   tzinfo-data
+  webrick
```

Comment:
Jekyll should probably include this for Ruby 3+. But fine, we can add it (even on 2.6+, doesn't hurt).

Reply:
Jekyll v4.3+ will include webrick as a dependency.